libraries/AP_HAL_ChibiOS/hwdef/ARK_CANNODE/hwdef.dat

```diff
@@ -6,7 +6,7 @@
 MCU STM32F4xx STM32F412Rx
 
 # board ID for firmware load
```

Comment:
I think we should add the words "see Tools/AP_Bootloader/board_types.txt" to the comment in each of these hwdef.dat, as I know people get confused about where these come from